satellite/repair/repair_test.go

```diff
@@ -368,36 +369,36 @@ func TestCorruptDataRepair_Succeed(t *testing.T) {
 			if nodesToKill[node.ID()] {
 				err = planet.StopPeer(node)
 				require.NoError(t, err)
-				_, err = satellite.Overlay.Service.UpdateUptime(ctx, node.ID(), false)
+				_, err = sat.Overlay.Service.UpdateUptime(ctx, node.ID(), false)
 				require.NoError(t, err)
 			}
 		}
 		require.NotNil(t, corruptedNode)
 
 		corruptPieceData(ctx, t, planet, corruptedNode, corruptedPieceID)
 
-		overlay := planet.Satellites[0].Overlay.Service
-		node, err := overlay.Get(ctx, corruptedNodeID)
+		over := planet.Satellites[0].Overlay.Service
+		node, err := over.Get(ctx, corruptedNodeID)
 		require.NoError(t, err)
 		corruptedNodeReputation := node.Reputation
 
-		satellite.Repair.Checker.Loop.Restart()
-		satellite.Repair.Checker.Loop.TriggerWait()
-		satellite.Repair.Checker.Loop.Pause()
-		satellite.Repair.Repairer.Loop.Restart()
-		satellite.Repair.Repairer.Loop.TriggerWait()
-		satellite.Repair.Repairer.Loop.Pause()
-		satellite.Repair.Repairer.Limiter.Wait()
+		sat.Repair.Checker.Loop.Restart()
```

Comment:
nit `sat` -> `satellite`

Reply:
@VinozzZ i on purpose renamed all vars to avoid package shadowing!
It’s it good to have vars named the same as imported packages.

Reply:
@stefanbenten Could you point where this is a problem in this file?

Reply:
It is not a problem that it fails to operate. Its more like unclear usage.
We import the package named satellite and have a variable shadowing it instead.